File: flask_backend/app/typedefs/_query.py
from ariadne import gql, ObjectType, QueryType
from ariadne.asgi import GraphQLSchema

query_type_defs = gql("""
    type Query{
        person: Person
        people: [Person]  
        elastic(_review:String!): [Elastic]
        neo4j: [Neo4j]
        mongodb: [MongoDB]
    }
""")

File: graphql/server/typedefs-resolvers/typedefs-main/_query.js
const { gql} = require('apollo-server')

const typeDefs = `
type Query{
    people: [person]
    person(name:String!): person
    movies: [movie]
    movie(title:String!): movie
    elastic(review:String!): [es]
    neo4j(cypher:String!):[neo4j]
}
`
module.exports = typeDefs